addons/coupon/security/ir.model.access.csv

```diff
@@ -0,0 +1,9 @@
+id,name,model_id/id,group_id/id,perm_read,perm_write,perm_create,perm_unlink
+access_program_salesman,salesman,model_coupon_program,sales_team.group_sale_salesman,1,0,0,0
```

Comment:
please make 'visually pleasing' names for ACLs, as they are visible e.g. in Studio and in Technical menus.
Something like `Coupon Program (Salesperson)`, `Coupon Program (Manager)`, etc.

Reply:
note that i'm talking about the `name` field here, so the second item in every row